src/components/Table/components/TD/TD.style.ts

```diff
@@ -0,0 +1,38 @@
+import type { SerializedStyles, Theme } from '@emotion/react';
+import { css } from '@emotion/react';
+
+import type { RowSize, TableProps } from 'components/Table';
+
+/** @TODO replace all css with tokens */
+
+export const getMinHeight = (rowSize: RowSize) => (theme: Theme) => {
+  switch (rowSize) {
+    case 'md':
+      return theme.globals.sizing.get('13');
+    case 'lg':
+      return theme.globals.sizing.get('15');
+    default: /** sm and default */
+      return theme.globals.sizing.get('11');
+  }
+};
+
+export const tdContainer =
+  ({ rowSize }: Pick<TableProps<any>, 'rowSize'>) =>
+  (theme: Theme): SerializedStyles => {
+    return css`
+      height: ${getMinHeight(rowSize)(theme)};
+      padding: 8px 16px;
```

Comment:
this is not coming from any token?

Reply:
tokens will be added after #797 and the rest of the parts are merged